src/hotspot/share/cds/aotMappedHeapLoader.hpp

```diff
@@ -54,7 +54,7 @@ class AOTMappedHeapLoader : AllStatic {
 
   // Can this VM map archived heap region? Currently only G1+compressed{oops,cp}
```

Comment:
```suggestion
  // Can this VM map archived heap region? Currently only G1.
```

Reply:
Sorry, my mistake. Just compressed CP. Maybe just delete the comment, which violates

Rule 1: Comments should not duplicate the code

Reply:
Ashu just reminded me that we do relocate uncompressed oops when loading the CDS archive at runtime. We never got around to do that for uncompressed Klass pointers in object headers, and it was also never really necessary.

So you are right, the comment should either just be removed or explain the G1 restriction.